assets/helpers/flashSale.js

```diff
@@ -220,33 +220,109 @@ const Sales: Sale[] = [
       },
     },
   },
+  {
+    subscriptionProduct: 'Paper',
+    activeRegions: ['GBPCountries'],
+    startTime: new Date(2019, 1, 4).getTime(), // 4 Feb 2019
+    endTime: new Date(2019, 1, 18).getTime(), // 17 Feb 2019 (to finish at 0:00 in the morning)
+    duration: '3 months',
+    saleDetails: {
+      GBPCountries: {
+        promoCode: 'GCC80X',
+        intcmp: '',
+        price: 5.40,
+        discountPercentage: 0.50,
+        saleCopy: {
+          featuredProduct: {
+            heading: 'Paper subscriptions',
+            subHeading: 'Save up to 68% on your newspaper',
+            description: 'Find analysis, opinions, reviews, recipes and more with a subscription to The Guardian and The Observer. Subscribe now and save an additional 50% for your first three months.',
+          },
+          landingPage: {
+            heading: 'The Guardian newspaper subscriptions',
```

Comment:
Flash sale specific copy for subHeading and standfirst copy on the print landing page shouldn't actually get used assuming that the changes to the print page go out first.